src/libraries/Native/Unix/System.Security.Cryptography.Native/apibridge.h

```diff
@@ -27,6 +27,7 @@ int32_t local_RSA_meth_get_flags(const RSA_METHOD* meth);
 int32_t local_RSA_set0_crt_params(RSA* rsa, BIGNUM* dmp1, BIGNUM* dmq1, BIGNUM* iqmp);
 int32_t local_RSA_set0_factors(RSA* rsa, BIGNUM* p, BIGNUM* q);
 int32_t local_RSA_set0_key(RSA* rsa, BIGNUM* n, BIGNUM* e, BIGNUM* d);
+int local_RSA_test_flags(const RSA *r, int flags);
```

Comment:
I don't remember why, but the other local_s use int32_t in the header.  Probably good to match.